docs/data/material/components/autocomplete/GitHubLabel.js

```diff
@@ -19,23 +19,27 @@ const StyledAutocompletePopper = styled('div')(({ theme }) => ({
     fontSize: 13,
   },
   [`& .${autocompleteClasses.listbox}`]: {
-    backgroundColor: theme.palette.mode === 'light' ? '#fff' : '#1c2128',
+    backgroundColor: '#1c2128',
     padding: 0,
     [`& .${autocompleteClasses.option}`]: {
       minHeight: 'auto',
       alignItems: 'flex-start',
       padding: 8,
-      borderBottom: `1px solid  ${
-        theme.palette.mode === 'light' ? ' #eaecef' : '#30363d'
-      }`,
+      borderBottom: `1px solid  ${'#30363d'}`,
       '&[aria-selected="true"]': {
         backgroundColor: 'transparent',
       },
       [`&.${autocompleteClasses.focused}, &.${autocompleteClasses.focused}[aria-selected="true"]`]:
         {
           backgroundColor: theme.palette.action.hover,
         },
+      ...theme.applyStyles('light', {
+        borderBottom: `1px solid  ${' #eaecef'}`,
+      }),
     },
+    ...theme.applyStyles('light', {
+      backgroundColor: '#fff',
+    }),
```

Comment:
We should fix these instances to use `theme.applyStyles('dark')`

Reply:
I've fixed the entire codebase.